src/PrestaShopBundle/Service/DataProvider/Marketplace/ApiClient.php

```diff
@@ -64,7 +64,7 @@ public function __construct(
     public function setSslVerification($verifySsl)
     {
         $this->toolsAdapter->refreshCaCertFile();
-        $this->addonsApiClient->setDefaultOption('verify', $verifySsl);
+        //$this->addonsApiClient->setDefaultOption('verify', $verifySsl);
```

Comment:
and here it is: in the version of Guzzle required by the Bundle using PHP 7.1, there is no such way to refresh the `verify` option. Is it an issue @Quetzacoalt91?

I may rewrite the client entirely if required, but if it's minor I think we can merge and deprecate ApiClient::setSslVerification().

Reply:
I remember Windows users having issues with SSL check, and this will have to be reviewed with them.
I advise you to remove this line if it does not exist anymore with the new version, but keep the setter in mind in case we have to find another solution.

Reply:
ok, thanks for the information :)